src/lib/utils.ts

```diff
@@ -16,3 +16,8 @@ export const setDate = (
 ): TaxFormUserInput => {
   return { ...taxform, datum: date.toLocaleString('sk-sk') };
 };
+
+export const formatCurrency = (value: number | string): string => {
+  const numericValue = typeof value === 'string' ? parseFloat(value) : value;
+  return `${(numericValue || 0).toFixed(2).replace(/\B(?=(\d{3})+(?!\d))/g, ' ').replace('.', ',')} EUR`;
```

Comment:
Mozno by som ten regexp dal do nejakej variable s popisnym menom, lebo niekto ako ja, neobdarovany talentom na regexp, si len domyslam, co to robi :D

Reply:
To je pravda, vacsina regexpov je write-only 🤣